tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -1506,6 +1506,12 @@ constexpr bool test_format_string() {
     return true;
 }
 
+template <class charT>
+void test_gh_4316() {

```

Comment:
Like below, let's capture the bug title here, to help readers understand what this is testing.

Reply:
Thanks for reminder!